tests/Kernel/CorporateBlocks/CorporateBlocksTestBase.php

```diff
@@ -5,19 +5,161 @@
 namespace Drupal\Tests\oe_theme\Kernel\CorporateBlocks;
 
 use Drupal\Tests\oe_theme\Kernel\AbstractKernelTestBase;
+use Drupal\Tests\rdf_entity\Traits\RdfDatabaseConnectionTrait;
+use Symfony\Component\DomCrawler\Crawler;
 
 /**
  * Base class for corporate block Kernel tests.
  */
 abstract class CorporateBlocksTestBase extends AbstractKernelTestBase {
 
+  use RdfDatabaseConnectionTrait;
+
+  /**
+   * The config factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
   /**
    * Modules to enable.
    *
    * @var array
    */
   protected static $modules = [
+    'oe_content',
+    'rdf_entity',
+    'rdf_skos',
+    'oe_corporate_site_info',
     'oe_corporate_blocks',
   ];
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->setUpSparql();
+
+    $this->installConfig([
+      'rdf_entity',
+      'oe_corporate_site_info',
+      'oe_corporate_blocks',
+    ]);
+
+    $this->installEntitySchema('rdf_entity');
+    $this->installEntitySchema('skos_concept');
+    $this->installEntitySchema('skos_concept_scheme');
+
+    $graphs = [
+      'corporate_body' => 'http://publications.europa.eu/resource/authority/corporate-body',
+    ];
+    $this->container->get('rdf_skos.skos_graph_configurator')->addGraphs($graphs);
+
+    $this->configFactory = $this->container->get('config.factory');
+    $corporate_site_info = $this->configFactory->getEditable('oe_corporate_site_info.settings');
+    $corporate_site_info->setData([
+      'site_owner' => 'http://publications.europa.eu/resource/authority/corporate-body/ACP-EU_JA',
+      'content_owners' => ['http://publications.europa.eu/resource/authority/corporate-body/COMMU'],
+    ]);
+    $corporate_site_info->save();
```

Comment:
We shouldn't do this here, we should test with empty values as well, as these values will be initially empty. We should also make sure that cache is invalidated correctly.